src/openforms/formio/formatters/tests/files/kitchensink_printable_legacy_text.json

```diff
@@ -1,79 +1,77 @@
+    "BSN Multi": "111222333; 123456782",
+    "BSN Multi Empty": "",
+    "Signature": "?!? TODO ?!?",
     "Signature Empty": "",
-    "Signature Hidden": "",
     "Map": "52.373087283242505, 4.8923054658521945",
-    "Map Empty": "52.379648, 4.9020928"
+    "Map Empty": ""
```

Comment:
Als dit problematisch is door het gedrag van het kaartcompoonent dan lekker zo laten en oplossen in #1346

Reply:
Ik laat dit leeg in de fixture zodat die klopt met gewenste gedrag, en pas de legacy test aan zodat dit wordt toegevoegd en die test klopt.